src/inmanta/data/model.py

```diff
@@ -130,8 +130,8 @@ class AttributeStateChange(BaseModel):
     Changes in the attribute
     """
 
-    current: Optional[Union[SimpleTypes, List[SimpleTypes], Dict[str, SimpleTypes]]] = None
-    desired: Optional[Union[SimpleTypes, List[SimpleTypes], Dict[str, SimpleTypes]]] = None
+    current: Optional[Any] = None
```

Comment:
or object?

Reply:
Alas, pydantic does not support object because it has no validators for it.